docs/SOPs/Managed_access/Review_and_export_SOP.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Data and metadata review and export SOP
+parent: Managed access
+grand_parent: SOPs
+nav_order: 5
+last_modified_date: 30/05/2024
+---
+
+<script src="https://kit.fontawesome.com/fc66878563.js" crossorigin="anonymous"></script>
+
+# Managed access dataset - Data and metadata review and export SOP
+Once the the wrangler has verified that the data and metadata files were transferred correctly to the hca-util are they can  proceed with submission creation and validation
+
+## Submission review and validation
+Data and metadata in a managed access submission are considered sensitive information and must be kept encrypted. Unlike in the case of open access datasets, review for a managed access dataset must be done without ever downloading the metadata spreadsheet - or the data - to a non-encrypted location, like a laptop.
+1. Once the metadata spreadsheet is uploaded to the hca-util area a lambda function will automatically trigger the upload of the spreadsheet to the project in ingest.
+2. If the upload fails because ingest cannot process the spreadsheet, get in touch with the contributor to review their submission in a call. The contributor should share their screen and walk through the spreadsheet.
+3. Review the submission for metadata errors and communicate with the contributor to fix them. Ask the contributor to upload the improved spreadsheet to the hca-util area and repeat from step 1. 
+4. Sync data files directly from hca-util to ingest staging bucket \
+`$ hca-util sync s3://org-hca-data-archive-upload-prod/<submission-uuid>`
+5. Add ontology terms were necessary, for example for methods, species and developmental stage.
```

Comment:
Within  HCA Data Repository Ingest Service, add ontology terms where necessary, for example for methods, species and developmental stage.